extensions/vscode/vscode.wake

```diff
@@ -29,28 +29,29 @@ def installRO path =
     | runJob
     | getJobOutput
 
+def createStdlibJSON (stdlibSources: List Path): Result Path Error =
+    def formatPath path =
+        cat ("\t\t\"", (getPathName path | replace `share/wake/lib` ""), "\"",)
+    def formattedPaths = 
+        map formatPath stdlibSources
+    def content = 
+        cat ("\{\n\t\"files\": [\n", (catWith ",\n" formattedPaths), "\n\t]\n\}",)
+    write "{here}/stdlib.json" content
```

Comment:
I should probably point out that we do have native JSON handling so you don't have to deal with the pretty-printing yourself:
```suggestion
    def formatPath path =
        getPathName path | replace `share/wake/lib` ""
    def formattedPaths =
        map formatPath stdlibSources
    def content =
        JObject ("files" :-> JArray (map JString formattedPaths), Nil)
    write "{here}/stdlib.json" (prettyJSON content)
```

Reply:
Thanks!